assets/dev/js/frontend/handlers/base-carousel.js

```diff
@@ -135,9 +135,55 @@ export default class CarouselHandlerBase extends SwiperHandlerBase {
 			},
 		};
 
+		this.applyOffsetSettings( elementSettings, swiperOptions, slidesToShow );
+
 		return swiperOptions;
 	}
 
+	applyOffsetSettings( elementSettings, swiperOptions, slidesToShow ) {
+		const offsetSide = elementSettings.offset_sides;
+		const isNestedCarouselInEditMode = elementorFrontend.isEditMode() && 'NestedCarousel' === this.constructor.name;
+
+		if ( ! isNestedCarouselInEditMode && offsetSide && 'none' !== offsetSide ) {
+			const offset = elementSettings.offset_width.size,
+				fallbackSlideWidth = 200,
+				containerWidth = this.getDefaultElements().$swiperContainer[ 0 ].offsetWidth;
+
+			let slideWidth = this.getDefaultElements().$slides[ 0 ].offsetWidth;
+
+			// If the slide width is not available, assume a default width
+			if ( ! slideWidth ) {
+				slideWidth = fallbackSlideWidth;
+			}
+
+			if ( 'right' === offsetSide ) {
```

Comment:
is this a switch case ?
if so 
it will be nice to see it as 
Switch (foo){
  case  (a) :
    handleCaseA();
    break;
    .....
}